src/open_inwoner/components/templates/components/Header/Header.html

```diff
@@ -3,16 +3,16 @@
 {% get_solo "configurations.SiteConfiguration" as config %}
 
 {% accessibility_header request=request %}
-<header class="header">
+<header class="header" aria-label="Navigatie header">
     <div class="header__container">
-        {% logo src=config.logo.file|thumbnail_url:'logo' alt="" %}
+        {% logo src=config.logo.file|thumbnail_url:'logo' alt=config.logo.default_alt_text %}
 
         <div class="header__menu">
             <div class="header__menu-icon">
                 {% icon icon="menu" %}
             </div>
             <div class="header__submenu">
-                <nav class="header__actions">
+                <nav class="header__actions" aria-label="Zoek navigatie mobiel">
```

Comment:
I don't think mobile specific functionality applies to assistive technology? Can we use only one navigation structure for assistive technology and hide the other using aria-hidden="true"?